vcpkg/consume/install-locally-modified-package.md

```diff
@@ -0,0 +1,251 @@
+---
+title: "Tutorial: Install a locally modified dependency"
+description: This tutorial shows you how to modify a vcpkg port and install it using overlays"
+author: vicroms
+ms.author: viromer
+ms.date: 10/24/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+
+#CustomerIntent: As a vcpkg user, I want to learn how to modify a vcpkg provided dependency and install it
+---
+
+# Tutorial: Install a locally modified dependency
+This tutorial guides you on how to modified a vcpkg port using custom overlays.
+We recommend that you read the tutorial on [packaging a
+library](../get_started/get-started-packaging.md) before proceeding.
+
+In this tutorial, you will learn to:
+
+> [!div class="checklist"]
+> * [Create an overlay port](#1---create-an-overlay-port)
+> * [Get the port's source code](#2---get-the-ports-source-code)
+> * [Create a temporary Git registry](#3---create-a-temporary-git-registry)
+> * [Modify the necessary files](#4---modify-the-necessary-files)
+> * [Generate a patch file](#5---generate-a-patch-file)
+> * [Modify `portfile.cmake` to apply the patch file](#6---modify-portfilecmake-to-apply-the-patch)
+> * [Install your overlay port](#7---install-your-overlay-port)
+
+## Prerequisites
+
+* A terminal
+* vcpkg
+* [Git](<https://git-scm.com/downloads>)
+* A C++ compiler
+
+## 1 - Create an overlay port
+The first step is to create an [overlay port](../concepts/overlay-ports.md) of
+the package you want to modify.
+
+1 - Create a directory to hold your overlay ports
+
+The overlay ports directory can be created in any filesystem location of your
+choosing. In any step of this tutorial replace `$OVERLAY_LOCATION` with your
+chosen location.
+
+::: zone pivot="shell-powershell, shell-cmd"
+
+```powershell
+mkdir "$OVERLAY_LOCATION"
+```
```

Comment:
This will cause the cmd zone pivot to show the PowerShell box, which is a bit weird. You should use the "console" box so it's generic.